docs/running-workflows/supported-systems/cwl/index.md

```diff
@@ -1,3 +1,178 @@
 # CWL
 
 The Common Workflow Language standard originated in life science community.
+
+Latest REANA 0.8 release series uses ``cwltool==3.1.20210628163208``.
```

Comment:
This could be better a table.

Reply:
We could expand the intro afterwards, independently of conformance tests results.